lib/puppet/network/http/api/v3.rb

```diff
@@ -14,9 +14,13 @@ class Puppet::Network::HTTP::API::V3
     Environments.new(Puppet.lookup(:environments))
   end)
 
-  def self.routes
-    Puppet::Network::HTTP::Route.path(%r{/v3}).
+  def self.master_routes
+    Puppet::Network::HTTP::Route.path(%r{v3}).
         any.
         chain(ENVIRONMENTS, INDIRECTED)
   end
+
+  def self.ca_routes
+    Puppet::Network::HTTP::Route.path(%r{v1}).any.chain(INDIRECTED)
```

Comment:
This makes sense to me... my only question is where should the `indirected_routes.rb` file live?